frontend/src/pages/PersonalPage.jsx

```diff
@@ -120,43 +114,44 @@ function PersonalPage(props) {
             success = false;
         });
 
-        // if (success) {
-        //     const userNoteResponseJSON = await userNotesResponse.json();
-        //     setNotes([]);
-        //     if (userNoteResponseJSON.note_reference) {
-        //         for (const noteRef of userNoteResponseJSON.note_reference) {
-        //             setNotes((arr) => [...arr, noteRef]);
-        //         }
-        //         const pdfResponse = await fetch(
-        //             "http://localhost:3000/v1/notes/file/" +
-        //                 notes[0].note_reference,
-        //             {
-        //                 method: "GET",
-        //                 headers: {
-        //                     Authorization:
-        //                         "bearer " +
-        //                         window.localStorage.getItem("authToken"),
-        //                 },
-        //             }
-        //         ).catch((err) => {
-        //             console.error(err);
-        //             success = false;
-        //         });
+        if (success) {
+            const userNoteResponseJSON = await userNotesResponse.json();
+            setNotes([]);
+            if (userNoteResponseJSON.notes) {
+                for (const noteRef of userNoteResponseJSON.notes) {
+                    setNotes((arr) => [...arr, noteRef]);
+                }
```

Comment:
Again, I don't think you need to reset the notes array and the for loop can be simplified.

Reply:
Don't need to ``setNotes([])``

Reply:
Removed it.